compiler/bin-js_of_ocaml/compile.ml

```diff
@@ -65,6 +66,11 @@ let run
   | `Name _, _ -> ());
   List.iter params ~f:(fun (s, v) -> Config.Param.set s v);
   List.iter static_env ~f:(fun (s, v) -> Eval.set_static_env s v);
+  Config.Flag.(
```

Comment:
At that point, i'm not sure using `Config.Flag` is helping us. What about we just pass the target_env to Linker directly whenever we need it ?

Reply:
Certainly an option. There are many entrypoints. I implemented it here: https://github.com/ocsigen/js_of_ocaml/commit/e44cf049b83be7665e42181f72e2d4eaaca9ed8b. It's a bit much, so I didn't merge it into this branch yet.

Reply:
I like it a lot, please merge into your PR. I made a few comments. I'll do another review after you include the additional commit. Thanks for your patience and for being responsive.

Reply:
> Thanks for your patience and for being responsive.

I should be saying the same to you. No worries. You keep giving valid feedback, so I'm happy to iterate.